pandas/core/indexes/period.py

```diff
@@ -77,20 +77,25 @@ class PeriodDelegateMixin(DatetimelikeDelegateMixin):
 )
 class PeriodIndex(DatetimeIndexOpsMixin, Int64Index, PeriodDelegateMixin):
     """
-    Immutable ndarray holding ordinal values indicating regular periods in
-    time such as particular years, quarters, months, etc.
+    Holds ordinal time values.
 
-    Index keys are boxed to Period objects which carries the metadata (eg,
-    frequency information).
+    An immutable ndarray. Indicates regular periods in time such as particular years,
+    quarters, months, etc. Index keys are boxed to Period objects which carries the
+    metadata (eg, frequency information).
 
     Parameters
     ----------
+    **fields
```

Comment:
I'm guessing you are documenting this from the `__new__` method, but shouldn't be documenting that externally as that isn't something the user would interact with.

Can you see how that actually gets mapped to the individual class constructors? That's what should be documented

Reply:
Ok thanks. I documented only because the docstrings validator identified it as missing from the string and raised an error. If the user doesn't define the periodIndex using `ordinal` or `data` a range is built using start, end and periods or freq and the other optional arguments held in `fields` by passing them to periodArray._generate_range(). 

Should I include that all in the docstring here or somewhere else or does it not matter bc it's not relevant to usage?